src/runtime_src/core/common/xrt_graph.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2020, Xilinx Inc - All rights reserved
+ * Xilinx Runtime (XRT) Experimental APIs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+// This file defines shim level XRT Graph APIs.
+
```

Comment:
xrt_graph.h is a shim level API that must be implemented by shim level drivers supporting the API.  That's clear.  What's makes this not so clear is that the header file is under xrt/core/common;  I wondering if we should move it to runtime_src/include and just not export it to end-users.  A few other headers files under runtime_src/include are doing same and are not being exported.

Reply:
I think you mean runtime_src/core/include, right? I moved xrt_graph.h there